packages/scss/src/components/_button.scss

```diff
@@ -398,91 +402,100 @@
 // WRAPPER
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
+ul.button-group {
+	flex-wrap: wrap;
+}
+
 .button-group {
 	display: inline-flex;
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
+	gap: _theme('commons.divider.width');
+	border-radius: _component('button.border.radius');
+	overflow: hidden;
 
 	&:focus-within {
-		box-shadow: 0 0 0 4px _color("primary", "200");
-		border-radius: _component("button.border.radius");
+		box-shadow: 0 0 0 4px _color('primary', 200);
 	}
 
-	.button {
-		// @include buttonColoring("button.default-palette");
+	&.mod-outlined {
```

Comment:
on pourrait rajouter la class "mod-outline" sur les ".button" pour le legacy ?

Reply:
By now, we should probably remove all the remaning `mod-outline` (in a big release, as a breaking change). It's not long to search / replace and it would reduce our dist size

Reply:
I'd like that too. But maybe we can do it in two steps to make it easier.